### ISSUE TITLE
contactType would benefit from more standardization

### ISSUE THREAD

**Issue by Meteor0id** [schemaorg/schemaorg#1926]
**The issue:**
Currently ContactPoint accepts only type "text", but offers no suggestions how a contactType could be defined.
Google search uses contactType to index websites, and supports the following values case insensitive:
"customer support", "technical support", "billing support", "bill payment", "sales", "reservations", "credit card support", "emergency", "baggage tracking", "roadside assistance", "package tracking".

I am missing any support for an information point. NGO's and many other types of public interest organisations have no use for any of the categories Google Search is supporting.
But I can hardly blame Google for this. There is no standard nor guidence for contactType other than the provided examples.

**The proposed solution:**
Therefor I propose that next to "Text", schema.org also defines a few common types.
For starters:
"customer support", "technical support", "billing support", "bill payment", "sales", "reservations", "credit card support", "emergency", "baggage tracking", "roadside assistance", "package tracking", "public information", "legal representative", "press information", "tipline"

**The alternative doom senario:**
The alternative is for larger search engines to such as Google to come up with their own standards, with no guarantee they will be usefull to crawlers of others. Of course they still can come up with their own, because type "text" will remain supported, but atleast some standardized guidence will be provided.

**Comment by rvguha** [comment]
I think this is a good idea. And we always have text as the fallback

guha

**Comment by danbri** [comment]
I agree - this should be improved. I'm not a huge fan of the Google list, but it's a start. Are there any other sources of useful potential values? /cc @tilid @tmarshbing @nicolastorzec

**Comment by danbri** [comment]
Previous discussion: https://github.com/schemaorg/schemaorg/issues/238

**Comment by Meteor0id** [comment]
Just to be sure I would like to add that schemaorg/schemaorg#238 duscusses contactPoint as a whole. This ticket focusses on contactPointType.

Certainly related and overlapping, but not exactly a duplicate.

**Comment by danbri** [comment]
@Meteor0id - indeed. If it was an exact duplicate, we'd have closed this one. We did discuss the [lack of contactType consensus](https://github.com/schemaorg/schemaorg/issues/238#issuecomment-137279259) previously. Our general approach has been to start from what can be agreed, and improve things step by step. Initially we didn't have much experience or requirements around contact type value lists, so we left it open-ended (i.e. text).  But it seems like a good time to try to improve things!

**Comment by danbri** [comment]
Actually I think I'm remembering mail threads that happened at the same time. Anyway it has long been an area where - as you say - more standardization would be useful...
